### PR TITLE
Reports Hero

### PR DESCRIPTION
**Pull request by philipjohn**
Introduces:
* New pattern category "Wikimedia Reports"
* New pattern: "Report Hero"
* CSS variables for viewport widths, copied from the Shiro theme
* Noto Sans font for headings

The pattern includes an image, headings and content for the hero section of a report. It adapts to different viewport widths according to the design at breakpoints of 390px, 720px, 100px, 1200px and 2000px.

**Note:** I'm not happy with how the content section moves relative to the headings, so I'm especially interested in feedback on how we can better keep the gap between the headings and content.

Screenshot 2024-03-14 at 12 08 54
Screenshot 2024-03-14 at 12 12 27
Screenshot 2024-03-14 at 12 13 00

### CODE REVIEW

File: src/frontend.scss [hunk 1]
```diff
@@ -4,3 +4,8 @@ button:focus {
 	box-shadow: 0 0 0 2px rgba(0, 125, 250, 0.8);
 	border-radius: 2px;
 }
+
+@font-face {
+	font-family: 'Noto Sans';
+	src: url('../assets/fonts/NotoSans-VariableFont_wdth-wght.ttf');
```
Comment (kadamwhite) [inline src/frontend.scss:10]: I remember there being some confusion about the fonts. I thought we were using the system default font stack, and Notowas just what Ivan was using for designs… Or did we confirm that we do need to ship this font? Sorry if I missed some communication about this

Comment (philipjohn) [reply src/frontend.scss:10]: Confirmed on the call yesterday that Noto Sans is correct :)

File: src/patterns/hero/editor.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			display: inline-block;
+			height: 20px;
+			padding-right: 5px;
+			width: 20px;
+			background-repeat: no-repeat;
+		}
+
+		h3.wp-block-heading {
+			font-family: 'Noto Sans';
+			font-size: 35px;
```
Comment (kadamwhite) [inline src/patterns/hero/editor.scss:36]: Flagging the same note from the other PR, use `rem` (which is 16px) for all font-sizes (and omit units on `line-height`)

**Review by goldenapples — COMMENTED** [review]
I haven't looked through the other PRs, so this may be getting addressed elsewhere already.

The image isn't scaling up if an image is uploaded smaller than the screen width:

image

The content isn't wrapping around the header container when it breaks on mobile:
image

**Comment by philipjohn** [comment]
> The image isn't scaling up if an image is uploaded smaller than the screen width:

Looks like this just needed Full Width on the image, sorted in 55da39a

**Comment by goldenapples** [comment]
I did a little bit of work on this locally, but haven't pushed it up other than the one change to pattern markup I did. 

The fix for vertical alignment in the sections with background is implemented elsewhere (see https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/pull/22), but there is a lot of kind of hacky CSS styling here that's fighting against the block defaults, and I'd like to remove a bunch of the custom styling in this PR before merging it, so that we can stay closer to core's styles.

**Pull request merged**